### PR TITLE
Verge.com

### PR DESCRIPTION
**Pull request by GayathriMohan**
Hi @nishintn,

Please find the working implementation of verge.com.

### CODE REVIEW

File: js/verge.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    document.getElementById("nav-hidden-menu").style.transform= "translateX(0%)";
+}
+
+function closeNav() {
+    document.getElementById("nav-menu").style.transform = "translateX(-100%)";
+    document.getElementById("nav-hidden-menu").style.transform = "translateX(-100%)";
+
+}
+
+function openarticles()
```
Comment (mad-nishin) [inline js/verge.js]: Use camelCase for function names.

**Comment by mad-nishin** [comment]
@GayathriMohan ,
Use camelCase for fn names.
Also have proper commit messages.

**Comment by GayathriMohan** [comment]
Hi @nishintn ,

I have made the necessary changes and updated the new source code.

**Comment by GayathriMohan** [comment]
Hi @nishintn ,

The JavaScript function naming convention has been changed and updated.

**Pull request merged**